pytorch_lightning/plugins/training_type/ddp.py

```diff
@@ -177,7 +180,19 @@ def set_world_ranks(self):
         self.global_rank = self.node_rank * self.num_processes + self.local_rank
         self.world_size = self.num_nodes * self.num_processes
 
+    def pre_configure_ddp(self):
+        # todo: PyTorch 1.7.0 DDP introduces ``self.reducer._rebuild_buckets()`` breaking manual_optimization
```

Comment:
is this an issue with lightning's manual optimization or with the pytorch implementation?

Reply:
This is an issue with PyTorch having a flag that seems to not completely disable the experimental feature despite it should!

Reply:
Hey @ananthsub. I would like to catch up on this issue.